pnpm-lock.yaml

```diff
@@ -682,6 +707,19 @@ packages:
       '@types/react':
         optional: true
 
+  '@radix-ui/react-toast@1.2.1':
+    resolution: {integrity: sha512-5trl7piMXcZiCq7MW6r8YYmu0bK5qDpTWz+FdEPdKyft2UixkspheYbjbrLXVN5NGKHFbOP7lm8eD0biiSqZqg==}
+    peerDependencies:
+      '@types/react': '*'
+      '@types/react-dom': '*'
+      react: ^16.8 || ^17.0 || ^18.0 || ^19.0 || ^19.0.0-rc
+      react-dom: ^16.8 || ^17.0 || ^18.0 || ^19.0 || ^19.0.0-rc
```

Comment:
**Suggestion:** Consider specifying a more restrictive version range for `react` and `react-dom` in the peer dependencies of `@radix-ui/react-toast`. Using `^19.0.0-rc` might unintentionally allow future release candidates that could introduce breaking changes or incompatibilities. [possible issue, importance: 8]
```suggestion
  '@radix-ui/react-toast@1.2.1':
    peerDependencies:
      react: ^19.0.0-rc-f994737d14-20240522
      react-dom: ^19.0.0-rc-f994737d14-20240522
```